Package.swift

```diff
@@ -16,18 +16,18 @@ let package = Package(
   ],
   dependencies: [
     .package(
-      url: "https://github.com/industrialbinaries/OpenSSL",
-      .branch("feature/asn1")
+      url: "https://github.com/industrialbinaries/ASN1Decoder",
+      .branch("feature/doc")
```

Comment:
Why we don't use `master`?

Reply:
It's older PR, last will be using `master` after marge `feature/refactoring` on `ASN1Decoder`